coops-ui/src/components/Dashboard/DirectorListSm.vue

```diff
@@ -1,71 +1,114 @@
 <template>
-  <ul class="list list-container">
-    <li class="list-item" v-for="director in directors" v-bind:key="director.id">
-      <v-avatar color="primary" size="25">
-        <span class="white--text small">{{ director.officer.firstName.substring(0,1)}}</span>
-      </v-avatar>
-      <div class="director-info">
+  <v-expansion-panels class="list list-container" :accordion=true :multiple=true>
+    <v-expansion-panel class="address-panel" v-for="director in directors" v-bind:key="director.id">
+      <v-expansion-panel-header>
+        <v-avatar color="primary" size="25">
+        <span class="">{{ director.officer.firstName.substring(0,1)}}</span>
+        </v-avatar>
         <div class="list-item__title">{{ director.officer.firstName }} {{ director.officer.lastName }}</div>
-        <div class="list-item__subtitle">
-          <ul class="address-details">
-            <li>{{ director.deliveryAddress.streetAddress }}</li>
-            <li>{{ director.deliveryAddress.addressCity }} {{ director.deliveryAddress.addressRegion }}
-              &nbsp;&nbsp;{{ director.postalCode}}</li>
-            <li>{{ director.deliveryAddress.addressCountry }}</li>
-            </ul>
-        </div>
-      </div>
-    </li>
-  </ul>
+      </v-expansion-panel-header>
+      <v-expansion-panel-content class="list-container-content">
+        <li class="list-item" >
+          <div class="director-info">
+            <div class="list-item_title">Delivery Address</div>
+            <div class="list-item__subtitle">
+              <ul class="address-details">
+                <li>{{ director.deliveryAddress.streetAddress }}</li>
+                <li>{{ director.deliveryAddress.addressCity }} {{ director.deliveryAddress.addressRegion }}
+                  {{ director.deliveryAddress.postalCode }}</li>
+                <li>{{ director.deliveryAddress.addressCountry }}</li>
+              </ul>
+            </div>
+          </div>
+        </li>
+        <li class="list-item" v-if="entityFilter(EntityTypes.BCorp)">
+          <div class="director-info">
+            <div class="list-item_title">Mailing Address</div>
+            <div class="list-item__subtitle">
+              <span v-if="isSameAddress(director.deliveryAddress, director.mailingAddress)">
+                Same as above
```

Comment:
Shouldn't this be the same text as Directors.vue:207?

Reply:
I definitely agree to an extent though the context is a little bit different, i had done this as per the designs :) I will mention it to them!

Reply:
Not changed.

Reply:
Sorry, i left it because it was the verbiage outlined in the designs :/